app/routes/questions/search.tsx

```diff
@@ -0,0 +1,19 @@
+import type {LoaderFunction} from '@remix-run/cloudflare'
+
+export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
+  const url = new URL(request.url)
+  const question = url.searchParams.get('question')
+
+  if (!question) return []
+
+  return await search(question)
+}
+
+export function search(question: string) {
+  const url = `${NLP_SEARCH_ENDPOINT}/api/search?query=${question}`
```

Comment:
@ccstan99 is this the right endpoint to use for this?

Reply:
Yes, but we should probably filter out already "Live on Site" questions shown in the previous step. I still need to add that parameter option for you.

Reply:
let me know when it's ready. Currently it just manually filters them out, which will result in various lengths of displayed questions

Reply:
Unfortunately, I probably won't get to this until the weekend. But I'll definitely let you know!